itests/kit2/node_opts_nv.go

```diff
@@ -7,12 +7,12 @@ import (
 	"github.com/filecoin-project/lotus/node"
 )
 
-func LatestActorsAt(upgradeHeight abi.ChainEpoch) node.Option {
+func LatestActorsAt(upgradeHeight abi.ChainEpoch) NodeOpt {
 	// Attention: Update this when introducing new actor versions or your tests will be sad
 	return NetworkUpgradeAt(network.Version13, upgradeHeight)
 }
 
-func NetworkUpgradeAt(version network.Version, upgradeHeight abi.ChainEpoch) node.Option {
+func NetworkUpgradeAt(version network.Version, upgradeHeight abi.ChainEpoch) NodeOpt {
```

Comment:
Makes does sense as it makes things more concise. But I think we'll have to revert it eventually because it removes the possibility of accompanying this with multiple `ConstructorOpts`, unless we make those mergeable.

Reply:
I would feel better about reverting this now and being OK with the slightly longer-hand form.

Reply:
Makes sense 👍